python/lsst/cp/pipe/ptc/cpPtcExtract.py

```diff
@@ -62,7 +62,7 @@ class PhotonTransferCurveExtractConnections(pipeBase.PipelineTaskConnections,
         deferLoad=True,
     )
     taskMetadata = cT.Input(
-        name="isr_metadata",
+        name="cpPtcIsr_metadata",
```

Comment:
This won't work with the new pipelines, or with all the verification stuff, I think.  I _think_ what we can do is define the connection here so that it can be overridden by somebody with an old config, and then del the connection in the init below so it never actually gets used/connected by the graph generator.